.rubocop_todo.yml

```diff
@@ -0,0 +1,254 @@
+# This configuration was generated by
```

Comment:
There were quite a few lint violations initially. I noticed that vets-api has this sort of file as well so I went this route, also to ensure this PR doesn't get too terribly large.